WORKSPACE.bazel

```diff
@@ -26,9 +26,9 @@ http_archive(
     ],
 )
 
-LLVM_COMMIT = "c2c49f464435fa6176c91898a94d8cadbdf93dfc"
+LLVM_COMMIT = "98f5a340975bc00197c57e39eb4ca26e2da0e8a2"
```

Comment:
Also, let's split the LLVM bump and preparatory changes for the 0.10.0 release? They are independent, right?